packages/models/src/classMap.js

```diff
@@ -81,9 +57,27 @@ export default class ClassMap {
   }
 
   codeObjectFromEvent(event) {
-    return this.codeObjects.find(
-      (co) => co.id.indexOf(event.toString()) !== -1
-    );
+    let codeObject;
+    // These types of events should not be reporting path and lineno, but sometimes
+    // they do.
+    if (!(event.httpServerRequest || event.httpClientRequest || event.sql)) {
```

Comment:
This is a point of weakness in the code; if we add new event types, we may need to add them to this exclusion clause.